packages/shared-influx-pieces/src/listeners/influxResourceAnalyticsSync.ts

```diff
@@ -0,0 +1,38 @@
+import type { PieceContext } from '@skyra/http-framework';
+import { Point } from '@skyra/influx-utilities';
+import { AnalyticsListener } from 'lib/AnalyticsListener.js';
+import { Points, Tags, Actions } from 'lib/AnalyticsSchema.js';
+import { cpus } from 'os';
+
+export class UserAnalyticsEvent extends AnalyticsListener {
+	public constructor(context: PieceContext, options: AnalyticsListener) {
+		super(context, { ...options, event: 'resourceAnalyticsSync' });
+	}
+
+	public run() {
+		this.writeCoreUsagePoint();
+		this.writeMemoryUsagePoint();
+		this.flush();
+	}
+
+	private writeCoreUsagePoint() {
+		const point = new Point(Points.PerCoreCPULoad) //
+			.tag(Tags.Action, Actions.Sync);
+
+		for (const [index, { times }] of cpus().entries()) {
+			point.floatField(`cpu_${index}`, (times.user + times.nice + times.sys + times.irq) / times.idle);
+		}
+
+		return this.writePoint(point);
```

Comment:
We don't read the returned value nor await (I think it's sync?) it, we don't need `return`:
```suggestion
		this.writePoint(point);
```